initfiles/componentfiles/configxml/thor.xsl

```diff
@@ -206,6 +206,10 @@
           </xsl:if>
         </xsl:for-each>
       </SSH>
+      <System>
```

Comment:
This file is not being changed....  We'll need to wait until thor uses the new configuration file.